### DISCUSSION TITLE
sms/email will not send

### DISCUSSION THREAD

**Discussion by cummingsk05** [jef/streetmerchant#2470] [Help]
Trying to set up sms/email alerts and I get this error when running the notification test.

`error :: ✖ couldn't send email Invalid login: 534-5.7.9 Application-specific password required. Learn more at
534 5.7.9  https://support.google.com/mail/?p=InvalidSecondFactor q84sm3066441qke.13 - gsmtp`

I have the dotenv file setup with my email address and pw but it's not sending to my gmail. Also get a similar message with sms.

`error :: ✖ couldn't send sms to [phone number ]for carrier tmobile Invalid login: 534-5.7.9 Application-specific password required. Learn more at
534 5.7.9  https://support.google.com/mail/?p=InvalidSecondFactor q126sm2917553qkd.48 - gsmtp`

**Comment by tkoeck** [comment]
This needs a  2nd factor authentication on the google side. You cannot use gmail for it I think. Use a SMTP server that has a simple authentication method. For the SMS part no idea but it also seems to want an authentication.

**Comment by ArmaanCPro** [comment]
So I tried fiddling around with all my settings. The finale was that I created a new email account, just for this purpose. In the Gmail settings, I turned on IMAP and POP. (Maybe it was called PUP not sure though) and in GOOGLE ACCOUNT settings I enabled allowing less secure apps to access my Gmail account. I enabled IMAP and POP on my main account, too.

**Reply by helmsley99** [reply]
what did you put in the field for stmp address and port number?
